pkgs/development/python-modules/setuptools/default.nix

```diff
@@ -27,6 +28,13 @@ let
     patches = [
       ./tag-date.patch
       ./setuptools-distutils-C++.patch
+      # Use sysconfigdata to find headers. Fixes cross-compilation of extension modules.
+      (fetchpatch {
```

Comment:
Is fetchpatch allowed here? Think this is somewhere in the bootstrap, hence causing the eval failure?

Reply:
I was wondering about that as well. I'm not sure how to know whether that is the case.

Reply:
The Python interpreter is a bootstrap package, but setuptools is not (or should not) be.